lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but the maximum allowed is 0.",
```

Comment:
Yes, I wrote it in #6192 to not break [this existing test](https://github.com/eslint/eslint/blob/4126f12f792231c844859e13a3fbd6b7ff78c4ee/tests/lib/rules/max-statements-per-line.js#L120).